### PR TITLE
Migrate to new cache API

### PR DESCRIPTION
**Pull request by jgavris**
Routes now need to differentiate between ActionCache and Content-Aware Storage artifacts.

### CODE REVIEW

**Comment by kersson** [comment]
What's the difference between the two routes?  Feel free to just point me to docs.

**Comment by jgavris** [comment]
It's not really documented yet :) 

https://github.com/bazelbuild/bazel/blob/master/src/main/java/com/google/devtools/build/lib/remote/README.md

Implementation of client is here:

https://github.com/bazelbuild/bazel/blob/master/src/main/java/com/google/devtools/build/lib/remote/blobstore/RestBlobStore.java#L33

File: Sources/bazel-rest-cache/Cache.swift [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+        let formatter = ByteCountFormatter()
+        formatter.allowedUnits = [
+            .useKB,
+            .useMB,
+            .useGB,
+        ]
+        return formatter
+    }()
+
+    private func addRoutes(router: Router) {
```
Comment (jgavris) [inline Sources/bazel-rest-cache/Cache.swift]: @kersson All I really did was refactor this into a class, and created identical sets of routes for each 'prefix'.

**Review by kersson — APPROVED** [review]
Didn't give it the most thorough review, but cool stuff 😁

**Pull request merged**